base_images/ubuntu_base-setup.sh

```diff
@@ -24,9 +24,14 @@ PKGS=( \
     git
     openssh-client
     openssh-server
+    rng-tools5
     software-properties-common
 )
 
+if ! ((CONTAINER)); then
+    $SUDO systemctl enable rngd
+fi
+
```

Comment:
This section needs to move down below `$SUDO apt-get -qq -y install "${PKGS[@]}"`

Reply:
PTAL.